src/preferenceCenter/preferenceCenter.commands.js

```diff
@@ -0,0 +1,183 @@
+import Logger from 'grapesjs-preset-mautic/src/logger';
+import PreferenceCenterService from 'grapesjs-preset-mautic/src/preferenceCenter/preferenceCenter.service';
+
+export default class PreferenceCenterCommands {
+  editor;
+
+  dcPopup;
+
+  pcService;
+
+  logger;
+
+  constructor(editor) {
+    this.pcService = new PreferenceCenterService(editor);
+    this.logger = new Logger(editor);
+  }
+
+  launchPreferenceCenterPopup(editor, sender, options) {
+    // Transform DC slots to token
+    // editor.runCommand('preset-mautic:dynamic-content-slots-to-tokens');
+
+    this.showPreferenceCenterEditor(editor, options);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  stopPreferenceCenterPopup(editor) {
+    editor.runCommand('preset-mautic:dynamic-content-tokens-to-slots');
+  }
+
+  /**
+   * Convert all dynamic content tokens on the canvas to
+   * human readable texts/slots/component.
+   * {dynamiccontent} => Dynamic Content
+   */
+  convertPreferenceCenterTokensToSlots() {
+    const components = this.pcService.getDcComponents();
+
+    components.forEach((comp) => {
+      this.pcService.transformDcTokenToSlot(comp);
+    });
+  }
+
+  /**
+   * Convert dynamic content slots to tokens
+   * Dynamic Content => {preferenceCenter}
+   *
+   * @param editor
+   * @returns integer nr of dynamic content slots found
+   */
+  // eslint-disable-next-line class-methods-use-this
+  convertPreferenceCenterSlotsToTokens(editor) {
+    // get all dynamic content elements loaded in the editor
+    const preferenceCenter = editor.DomComponents.getWrapper().find('[data-slot="preferenceCenter"]');
+    preferenceCenter.forEach((preferenceCenter) => {
+      const attributes = preferenceCenter.getAttributes();
+      const decId = attributes['data-param-dec-id'];
+      // If it's not a token -> convert to token
+      if (decId >= 0) {
+        const dynConId = PreferenceCenterCommands.getPcStoreId(attributes['data-param-dec-id']);
```

Comment:
please rename all the old code like `dynCon` to something that identifies the preference center